### PR TITLE
Serverside API for mockserver

### PR DESCRIPTION
**Pull request by sqren**
It should be possible to perform the same actions (addMock, removeListener, getRequests etc.) when using mockServer as a module vs. using it through the client (over http).

This does not contain any breaking changes.

@Tradeshift/collaboration

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
     "start": "./bin/http-mockserver.js",
     "debug": "node --inspect --debug-brk src/server/index.js",
     "lint": "eslint ./src",
-    "test": "jest"
+    "test": "NODE_ENV=test jest"
```
Comment (tstapleton) [inline package.json]: i thought jest automatically sets this (the note in http://facebook.github.io/jest/docs/getting-started.html#additional-configuration)

Comment (sqren) [inline package.json]: That's odd. Without this `process.env.NODE_ENV` was `undefined`

Comment (sqren) [inline package.json]: Just verified, and it's not necessary. Thanks!

**Comment by gogozby** [comment]
Good job. 💯  / 💯

**Comment by gogozby** [comment]
@sqren Btw, do you need to bump the version for this change?

**Comment by sqren** [comment]
Done :)

**Pull request merged**